main.rb

```diff
@@ -15,6 +16,11 @@
       config.sessions = { default: { uri: 'mongodb://localhost:27017/saywut' } }
     end
   end
+  Slack::Post.configure(
+    webhook_url: 'https://hooks.slack.com/services/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' ,
```

Comment:
Line is too long. [98/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.